src/parser/jobs/rdm/modules/ManaStackGauge.tsx

```diff
@@ -0,0 +1,214 @@
+			return
+		}
+
+		if (this.manaStackGauge.capped) {
+			const manaSpender = this.gauge.spenderModifiers.get(event.action)
+			this.debug(`Gauge is Broken by action ${this.data.getAction(event.action)?.name}`)
+			this.debug(`manaSpender is ${manaSpender} white mana value of ${manaSpender?.white} and black mana value of ${manaSpender?.black}`)
+			this.debug(`Time the Overcap happened ${this.parser.formatEpochTimestamp(event.timestamp)}`)
+			if (manaSpender != null) {
+				this.manaStatistics.white.overcapLoss += Math.abs(manaSpender.white)
+				this.manaStatistics.black.overcapLoss += Math.abs(manaSpender.black)
+			}
+		}
+
+		this.manaStackGauge.modify(modifier.manaStack)
+	}
+
+	private onGaugeSpender(event: Events['action']) {
+		const modifier = this.spenderModifiers.get(event.action)
+
+		if (modifier == null) {
+			if (this.shouldGaugeBreak(event)) {
+				this.onGaugeBreak(event)
+			}
+			return
+		}
+
+		this.manaStackGauge.modify(modifier.manaStack)
+	}
+
+	private onGaugeBreak(event: Events['action']) {
+		const breaker = {} as GaugeBreaker
+		breaker.action = event.action
+		breaker.timestamp = event.timestamp
+		breaker.manaStacksLost = this.manaStackGauge.value
+		this.gaugeBreak.push(breaker)
+		this.manaStackGauge.reset()
+	}
+
+	private shouldGaugeBreak(event: Events['action']) {
+		this.debug(`manaStackGauge is ${this.manaStackGauge.value} and the action is ${this.data.getAction(event.action)?.name} with id ${event.action} and it's included? ${this.gaugeBreakers.includes(event.action)}`)
+		return this.gaugeBreakers.includes(event.action) && this.manaStackGauge.value > MINIMUM
+	}
+
+	private onComplete() {
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.RESOLUTION.icon,
+			content: <Fragment>
+				<Trans id="rdm.manaStackGauge.suggestions.manaStack-wasted-content">Ensure you don't overcap your ManaStacks before using your finisher combo; overcapping ManaStacks means you used more than 3 enchanted skills in a row.</Trans>
+			</Fragment>,
+			tiers:  this.severity,
+			value:  this.manaStackGauge.overCap,
+			why: <Fragment>
+				<Trans id="rdm.manaStackGauge.suggestions.manaStack-wasted-why">You lost { this.manaStackGauge.overCap} ManaStacks due to capped Gauge resources</Trans>
+			</Fragment>,
+		}))
+
+		const manaStacksLost = this.gaugeBreak.reduce((sum, current) => sum + current.manaStacksLost, 0)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.RESOLUTION.icon,
+			content: <Fragment>
+				<Trans id="rdm.manaStackGauge.suggestions.manaStack-loss-content">Ensure that you don't use a non Enchanted GCD when you have ManaStacks, otherwise you lose them all</Trans>
+			</Fragment>,
+			tiers:  this.severity,
+			value:  manaStacksLost,
+			why: <Fragment>
+				<Trans id="rdm.manaStackGauge.suggestions.manaStack-loss-why">You lost { manaStacksLost} ManaStacks due to using a non Enchanted GCD</Trans>
+			</Fragment>,
+		}))
+
+		this.statistics.add(new DualStatistic({
```

Comment:
Should we suppress this statistic if there was no overcap loss (and thus no suggestion being generated?  Alternatively, can we roll this statistic into the Why of the overcap suggestion?

Reply:
We don't currently suppress the manafication version, and I haven't been asked to, so I treated this one the same; since if I suppress it here I should suppress it there.  We're currently not even sure if this particular metric will stay, but was deemed potentially useful to see for now.  

That said I don't really have an opinion on whether it should or shouldn't; would you like this one suppressed for now if it has a zero value pair, until I can get additional feedback?

Reply:
I guess I don't have a strong opinion on it.  One thing to consider for the statistic as a whole is how bloated does it make the page on mobile to have this statistic (especially in the no waste case, but even if there is data here, is the data valuable enough to justify the extra space this takes up)?

Reply:
Lana's response when asked was that of no strong opinion and keeping it there would be more for consistency.  The usefulness of this and of the manafication metric are currently being debated, given the change to manafication.  We're honestly just not certain how much of a problem this particular item is, since this tracking of it is brand new.